tests/test_tag_to_span.py

```diff
@@ -0,0 +1,46 @@
+import pytest
+import spacy
+
+from presidio_evaluator import tag_to_span
+
+
+@pytest.fixture()
+def spacy_en_core_web_sm():
+    return spacy.load("en_core_web_sm")
+
+
+@pytest.mark.parametrize(
+    "text, tags, ents",
+    [
+        ("Dan  is my name. ", ["B-name", "O", "O", "O", "O"], [[0, 1, "name"]]),
+        ("Dan Brown lives here", ["B-name", "I-name", "O", "O", "O"], [[0, 2, "name"]]),
+        ("Dan Dan Dan", ["B-name", "B-name", "B-name"], [[0, 3, "name"]]),
+        ("Nothing to look for", ["O", "O", "O", "O"], []),
+        ("It's Dan", ["O", "O", "B-name"], [[2, 3, "name"]]),  # BIO tagging scheme
+        ("It's Dan", ["O", "O", "name"], [[2, 3, "name"]]),  # IO tagging scheme
+        ("It's Dan", ["O", "O", "B-name"], [[2, 3, "name"]]),  # wrong tagging scheme
+        (
+            "It's Dan Washington",
+            ["O", "O", "U-name", "U-loc"],
+            [[2, 3, "name"], [3, 4, "loc"]],
+        ),
+        (
+            "Dan is Washington",
+            ["U-name", "O", "U-loc"],
+            [[0, 1, "name"], [2, 3, "loc"]],
+        ),
+    ],
+)
+def test_tag_to_span(spacy_en_core_web_sm, text, tags, ents):
```

Comment:
It's great! The function works well. I also add the other test cases and all of them passed. Thanks @omri374 for working on it.

Reply:
Thanks!